### ISSUE TITLE
Add cost() Function

### ISSUE THREAD

**Issue by teamaffixmb-jake** [team-affix/aurora#66]
It would be a very nice addition to have a manual cost() function, built in to each model, to allow the user to quickly grab an estimate of how well their neural net performed in its predictions for the most recent training set.

**Comment by teamaffixmb-jake** [comment]
Instead of adding a cost() function for each model, the framework has been refactored so as to include individual cost function, which wrap the outside of models, and each loss function returns the loss as a double upon a cycle() call. Each loss function is itself a model in Aurora.
